### PR TITLE
Troubleshooting added for AWS Observability hierarchy

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by himsharma01 — APPROVED** [review]
lgtm

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md [hunk 1]
```diff
@@ -160,9 +161,7 @@ The below tables displays the response for each text box in this section.
 
 ## Step 12: Create stack
 
-1. In **Capabilities and transforms** click each checkbox.
-
-    ![CFT_Capabilities_Transforms.png](/img/observability/CFT_Capabilities_Transforms.png)
+1. In **Capabilities and transforms** click each checkbox.<br/><img src={useBaseUrl('img/observability/CFT_Capabilities_Transforms.png')} style={{border: '1px solid gray'}} alt="CFT_Capabilities_Transforms" width="800"/>
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md:164]:
```suggestion
1. Under **Capabilities and transforms**, click each checkbox.<br/><img src={useBaseUrl('img/observability/CFT_Capabilities_Transforms.png')} style={{border: '1px solid gray'}} alt="CFT_Capabilities_Transforms" width="800"/>
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md [hunk 2]
```diff
@@ -195,21 +190,11 @@ This section walks you through the process of troubleshooting an AWS CloudForma
...
+1. Look for direct reasons for the failure that is available in the parent stack, as shown in the following example.<br/><img src={useBaseUrl('img/observability/Troubleshooting_2.png')} style={{border: '1px solid gray'}} alt="Troubleshooting_2" width="700"/>
+1. To find indirect reasons for the failure, go to the nested stack mentioned in the status reason, as shown in the following example. Take a note of the resources mentioned in the reason.<br/><img src={useBaseUrl('img/observability/Troubleshooting_3.png')} style={{border: '1px solid gray'}} alt="Troubleshooting_3" width="700"/>
+1. Select the deleted option to find the nested stacks, as shown in the following example.<br/><img src={useBaseUrl('img/observability/Troubleshooting_4.png')} style={{border: '1px solid gray'}} alt="Troubleshooting_4" width="700"/>
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md:196]:
```suggestion
1. Select the deleted option to find the nested stacks, as shown in the following example.<br/><img src={useBaseUrl('img/observability/Troubleshooting_4.png')} style={{border: '1px solid gray'}} alt="Troubleshooting_4" width="500"/>
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md [hunk 3]
```diff
@@ -256,5 +241,21 @@ To uninstall the AWS Observability Solution:
 1. Log in to your AWS account and go to CloudFormation.
 1. Select the main stack you want to delete.
 1. Select **Delete**.
+<img src={useBaseUrl('img/observability/CFT_Uninstall.png')} style={{border: '1px solid gray'}} alt="CFT_Uninstall" width="800"/>
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md:244]:
```suggestion
1. Select **Delete**.<br/><img src={useBaseUrl('img/observability/CFT_Uninstall.png')} style={{border: '1px solid gray'}} alt="CFT_Uninstall" width="800"/>
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md [hunk 4]
```diff
@@ -256,5 +241,21 @@ To uninstall the AWS Observability Solution:
...
+
+    ```sql
+    account= mobilebankingprod | count by _collector , _source
+    ```
+    <br/><img src={useBaseUrl('img/observability/metric-query.png')} style={{border: '1px solid gray'}} alt="metric-query" width="800"/>
+1. Delete the source or remove the account tag from the same metric source. After this, the account will be automatically removed from the AWS Observability hierarchy in the next 24 hours.
+
+:::note
+Removing the account tag will not stop the metrics ingestion.
+:::
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md:260]:
```suggestion
1. Run the required metric query to identify from which source and collector data is getting ingested. For this example, enter the below metric query:
    ```sql
    account= mobilebankingprod | count by _collector , _source
    ```
    <br/>metric-query
1. Delete the source or remove the account tag from the same metric source. After this, the account will be automatically removed from the AWS Observability hierarchy in the next 24 hours.
    :::note
    Removing the account tag will not stop the metrics ingestion.
    :::
```

**Review by kimsauce — APPROVED** [review]
looks great - just a few edits

**Comment by JV0812** [comment]
Thanks for your review Kim.

**Pull request merged**